freshmaker/image_verifier.py

```diff
@@ -22,123 +22,99 @@
 # Written by Jan Kaluza <jkaluza@redhat.com>
 
 from freshmaker import conf
-from freshmaker.lightblue import LightBlue
+from freshmaker.pyxis_gql import PyxisGQL
+from mypy_extensions import TypedDict
+from typing import Dict, List
 
 
-class ImageVerifier(object):
+class DataElements(TypedDict):
+    repository: Dict[str, List[str]]
```

Comment:
The standard python types for `dict`, `list`, `tuple`, etc... support element hinting ever since python 3.9. The typing lib types `Dict`, `List`, ... are deprecated (and will be removed in some future version):
https://peps.python.org/pep-0585/

My suggestion is to change these type hints, just as in `config.py`.

Reply:
Good point, I've updated type hinting related to the modules in this PR.